File: Programming EF Code First/chapter4/Chapter 4 VB/Fluent/DataAccess/DestinationConfiguration.vb
Imports System.Data.Entity.ModelConfiguration
Imports Model

Namespace DataAccess
  Public Class DestinationConfiguration
	  Inherits EntityTypeConfiguration(Of Destination)
	Public Sub New()
      Me.Property(Function(d) d.Name).IsRequired()

      Me.Property(Function(d) d.Description).HasMaxLength(500)

      Me.Property(Function(d) d.Photo).HasColumnType("image")
	End Sub
  End Class
End Namespace

File: Programming EF Code First/chapter3/Chapter3/BAGA_FluentAPI/DataAccess/DestinationConfiguration.cs
using System.Data.Entity.ModelConfiguration;
using Model;

namespace DataAccess
{
  public class DestinationConfiguration : EntityTypeConfiguration<Destination>
  {
    public DestinationConfiguration()
    {
      Property(d => d.Name)
        .IsRequired();

      Property(d => d.Description)
        .HasMaxLength(500);

      Property(d => d.Photo)
        .HasColumnType("image");
    }
  }
}